blueprints/quarkus/001_installing_jhipster_quarkus.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Installing JHipster Quarkus
+sitemap:
+priority: 0.5
+lastmod: 2021-01-07T08:40:00-00:00
+---
+
+# Installing JHipster Quarkus
+
+Before using JHipster Quarkus, you have to prepare your environment (Java, Node) and install JHipster.
```

Comment:
maybe, you should mention that the JHipster version should be the same than the one supported by the blueprint ?